### PR TITLE
Added operation HSigmoid-5

### PR DESCRIPTION
**Pull request by a-sidorova**
Added operation HSigmoid-5

### CODE REVIEW

**Review by antonvor — APPROVED** [review]

**Review by antonvor — APPROVED** [review]

File: src/common/math_utils.hpp [hunk 1]
```diff
@@ -310,6 +310,13 @@ inline U mish_fwd(T s) {
     return (U)(s * ::tanhf(v));
 }
 
+template <typename T,
+        typename U = typename utils::remove_reference<T>::type>
+inline U hsigmoid_fwd(T s) {
+    float v = s + 3.0f;
+    return (U)(v < 0.0f ? 0.0f : v > 6.0f ? 1.0f : v / 6.0f);
```
Comment (dmitry-gorokhov) [inline src/common/math_utils.hpp:317]: Maybe I missed smt by according to [the spec](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/activation/HSigmoid_5.md) this formula is not correct. Should be similar to `hswish_fwd` but w/o final multiplication on src:
```suggestion
float v = s + 3.0f;
v = v > 0.0f ? v : 0.0f;
v = v < 6.0f ? v : 6.0f;
return (U)(v / 6.0f);
```

Please correct me if I am wrong.

File: src/common/math_utils.hpp [hunk 2]
```diff
@@ -327,8 +334,9 @@ inline bool eltwise_fwd_preserves_zero(alg_kind_t alg, bool jit_impl = false) {
     using namespace utils;
     const bool preserves_zero = true
         && !one_of(alg, eltwise_linear, eltwise_soft_relu, eltwise_logistic,
-                eltwise_exp, eltwise_clamp, eltwise_not, eltwise_swish)
-        && IMPLICATION(jit_impl, !one_of(alg, eltwise_elu, eltwise_tanh, eltwise_clamp, eltwise_not, eltwise_swish));
+                eltwise_exp, eltwise_clamp, eltwise_not, eltwise_swish, eltwise_hsigmoid)
+        && IMPLICATION(jit_impl, !one_of(alg, eltwise_elu, eltwise_tanh, eltwise_clamp,
+                                             eltwise_not, eltwise_swish, eltwise_hsigmoid));
```
Comment (dmitry-gorokhov) [inline src/common/math_utils.hpp:339]: `eltwise_fwd_preserves_zero` should return true is f(0) = 0.  Does this condition works for HSigmoid?

**Review by dmitry-gorokhov — CHANGES_REQUESTED** [review]

File: src/common/math_utils.hpp (hunk 1 above)
Comment (a-sidorova) [reply src/common/math_utils.hpp]: It is the same. But I rewrote this function line by line like your sample. Thank you

File: src/common/math_utils.hpp (hunk 2 above)
Comment (a-sidorova) [reply src/common/math_utils.hpp]: Fixed, thank you!

File: src/common/math_utils.hpp (hunk 1 above)
Comment (dmitry-gorokhov) [reply src/common/math_utils.hpp]: Ok, I see. Anyway multi line implementation looks much more readable from my perspective.

**Review by dmitry-gorokhov — APPROVED** [review]

**Pull request merged**